### ISSUE TITLE
The Technique cards should explicitly say that they change your limit break

### ISSUE THREAD

**Issue by JuniorUdale** [TerryCavanagh/diceydungeons.com#2730]
e.g. Witch Technique felt weird to me, because at first glance it looked like I was spending 3 dice to get 3 new dice? Not sure if this would fit on the card though

**Comment by TerryCavanagh** [comment]
Yeah, hmm, not gonna fit without adding an extra line, I think... Maybe that's fine, though.
